### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by miccal**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -386,7 +386,7 @@ We maintain separate Taps for different types of binaries. Our nomenclature is:
 
 ### Stable Versions
 
-Stable versions live in the main repository at [caskroom/homebrew-cask](https://github.com/caskroom/homebrew-cask). They should run on the latest release of OS X or the previous point release (in 2015, for example, that meant El Capitan and Yosemite).
+Stable versions live in the main repository at [caskroom/homebrew-cask](https://github.com/caskroom/homebrew-cask). They should run on the latest release of OS X or the previous point release (in 2016, for example, that meant El Capitan and Yosemite).
```
Comment (vitorgalvao) [inline CONTRIBUTING.md]: [Nope](https://github.com/caskroom/homebrew-cask/pull/16304#discussion_r48694643).

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -282,7 +282,7 @@ cask 'lynkeos' do
   homepage 'http://lynkeos.sourceforge.net/'
   license :gpl
 
-  app "Lynkeos-App-#{version.dots_to_hyphens}/Lynkeos.app"
+  app "Lynkeos-App-#{version.gsub('.', '-')}/Lynkeos.app"
```
Comment (vitorgalvao) [inline CONTRIBUTING.md]: Why are you changing this?

File: CONTRIBUTING.md (hunk 1 above)
Comment (tsparber) [inline CONTRIBUTING.md]: Should be 2015, see https://github.com/caskroom/homebrew-cask/pull/16304 (comment on outdated diff)

File: CONTRIBUTING.md (hunk 2 above)
Comment (miccal) [inline CONTRIBUTING.md]: [So it matches the current version](https://github.com/caskroom/homebrew-cask/blob/master/Casks/lynkeos.rb)

Comment (vitorgalvao) [inline CONTRIBUTING.md]: Encouraging the new methods is important. Please change it back; I’ll correct the current version.

**Comment by miccal** [comment]
Fixed.

**Pull request merged**